docs/usage.rst

```diff
@@ -616,6 +616,62 @@ Thus you can use this to validate a document against several schemas without imp
 .. versionadded: 0.10
 
 
+excludes
+~~~~~~~~
+
+You can declare fields to excludes others:
+
+.. doctest::
+
+    >>> v = Validator()
+    >>> schema = {'this_field': {'type': 'dict',
+    ...                          'excludes': 'that_field'},
+    ...           'that_field': {'type': 'dict',
+    ...                          'excludes': 'this_field'}}
+    >>> v.validate({'this_field': {}, 'that_field': {}}, schema)
+    ... False
```

Comment:
that has to be

```
False
```

also true for all following output of returned objects. with leading dots it's interpreted as statement, not as expected result.